src/NanoporeMultiEventalign.jl

```diff
@@ -15,6 +15,7 @@ include("dtw.jl")
 
 
 export dtw
-export loadfasta, loadkmers, fasta_to_kmer_valus
+export loadnanoporefast5, loadfasta, loadkmers, fasta_to_kmer_valus
```

Comment:
should be values?
fasta_to_model maybe? 

After you fix this try merging this branch to the main, fix conflicts beforehand.